core/src/main/kotlin/io/specmatic/test/DataRecorder.kt

```diff
@@ -0,0 +1,21 @@
+package io.specmatic.test
+
+import io.specmatic.core.log.HttpLogMessage
+
+object DataRecorder {
+
+    val testHttpLogMessages = mutableListOf<HttpLogMessage>()
+    val stubHttpLongMessages = mutableListOf<HttpLogMessage>()
```

Comment:
I guess you meant to name this `stubHttpLogMessages`?

Reply:
thanks, fixed the typo